Library/Homebrew/cmd/update-report.rb

```diff
@@ -734,6 +778,16 @@ def dump_renamed_formula_report
     output_dump_formula_or_cask_report "Renamed Formulae", formulae
   end
 
+  def dump_renamed_cask_report
```

Comment:
Given https://github.com/Homebrew/brew/blob/c6e438b3912b95ef5d75de9626bd4d065f071999/Library/Homebrew/cmd/update-report.rb#L679-L680 this may be worth skipping and we can make a minor release sooner rather than later?

Reply:
I don't quite get why it shouldn't possible with the API, as long as you have a previous API JSON to compare against?

Reply:
I think we have the checksum of the formula/cask files.

Reply:
If we already have it: I wouldn't be opposed to it being implemented and undeprecated.